kuma/javascript/src/payments/components/subscription-form.jsx

```diff
@@ -247,47 +214,31 @@ export default function SubscriptionForm() {
                 // If window.mdn.triggerAuthModal is falsy, it most likely means
                 // it deliberately doesn't want this user to use a modal. E.g.
                 // certain mobile clients.
-                window.location.href = `/${locale}/users/account/signup-landing?next=${next}`;
+                window.location.assign(
+                    `/${locale}/users/account/signup-landing?next=${next}`
+                );
             }
         }
     }
 
     let content;
     if (formStep === 'server_error') {
         content = (
-            <section className="error">
```

Comment:
Noticed this and the error message below were similar so I thought I'd extract them into one component.